### PR TITLE
Update Capency documentation with the latest features

### PR DESCRIPTION
**Pull request by real34**
This PR improves the SmartForms documentation related to supporting implementations and Capency specifically.

It contains the latest configurations needed to configure the integration.

### CODE REVIEW

**Comment by real34** [comment]
I'm merging it so that I can send a link in support, but will handle PR reviews afterwards.

File: source/docs/advanced/features/smart-forms.md [hunk 1]
```diff
@@ -51,6 +64,12 @@ module.exports = {
 };
 ```
 
+Restart the project, and your existing fields should now use Capency webservices to provide suggestions and validate data.
+
+<blockquote class="info">
+  **ProTip:** you can use the `front-commerce:smart-forms:capency` value in your `DEBUG` environment variable to view more details about API interactions with Capency services.
```
Comment (dpobel) [inline source/docs/advanced/features/smart-forms.md:70]: this should also be mentioned in https://developers.front-commerce.com/docs/reference/environment-variables.html#Debugging (and maybe it's a good occasion that no other value was forgotten)

**Review by dpobel — COMMENTED** [review]
:+1:

File: source/docs/advanced/features/smart-forms.md (hunk 1 above)
Comment (real34) [reply source/docs/advanced/features/smart-forms.md:70]: Done in https://github.com/front-commerce/developers.front-commerce.com/pull/212

**Pull request merged**